src/org/rust/lang/core/parser/RustParserUtil.kt

```diff
@@ -29,6 +30,38 @@ public object RustParserUtil : GeneratedParserUtilBase() {
     //
 
     @JvmStatic
+    public fun injectInto(b: PsiBuilder, level: Int, s: Parser, t: Parser) : Boolean {
```

Comment:
Is this intentional that `t` is unused? 

I would prefer other name, because there is also `} catch (t: Throwable) {` inside.